### PR TITLE
issue 5071 fix

### LINKED ISSUE #5071: [$30] Clear filter does not clear the status filter option
Clear filter does not clear the status filter option. Clear filter must set the status option back to "Active"
Screenshot 2020-10-08 at 2 21 47 PM

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**
#5071

### CODE REVIEW

**Comment by luizrrodrigues** [comment]
@PrakashDurlabhji Please to right branch `bug-bash`

File: src/shared/components/challenge-listing/Filters/FiltersPanel/index.jsx [hunk 1]
```diff
@@ -433,6 +433,7 @@ export default function FiltersPanel({
               events: [],
               endDateStart: null,
               startDateEnd: null,
+              status: 'Active',
```
Comment (luizrrodrigues) [inline src/shared/components/challenge-listing/Filters/FiltersPanel/index.jsx:436]: When click in Clear Filters, status need change to `All`

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues updated

**Review by luizrrodrigues — CHANGES_REQUESTED** [review]
https://github.com/topcoder-platform/community-app/issues/5071#issuecomment-709588671

**Comment by PrakashDurlabhji** [comment]
@luizrrodrigues updatd

**Review by luizrrodrigues — APPROVED** [review]

**Pull request merged**